### ISSUE TITLE
Problem with crx file

### ISSUE THREAD

**Issue by dtantono** [FilipePS/Traduzir-paginas-web#227]
Hi
I tried with MS Edge Stable Version 91.0.864.70 (Official build) (64-bit) for Windows 10 and the browser blocked it and when I tried to reinstall by load the unpacked option via zip file. The browser thinks that this extension have a problem. Is this extension code signed? or is it a bug? 
<img width="834" alt="Screenshot_2021-07-18_22-15-13" src="https://user-images.githubusercontent.com/29157187/126068515-e0e9cd0f-26d1-49b0-b82a-40a22ec57310.png">

**Comment by FilipePS** [comment]
The CRX file was generated directly by the tool that chrome provides.
As it is not signed by a trusted source, the browser blocks the extension.
This "trusted source" would be the chrome store, edge, opera, etc...

**Comment by dtantono** [comment]
Understand, But, edge on my Linux box does not complain about this and since the largest market share is Windows and MS Edge is the default browser for Windows PC. I hope you can solve the problem soon.

**Comment by DarknessKiller123** [comment]
From what I know Chromium in Linux left the users use crx from other sources, and in Windows you can only install from web store. The problem for me was that the crx doesn't update and I have to update it manually.

**Comment by dtantono** [comment]
Yeah, that is weird Linux have better versatility than Windows despite Windows have a bigger market share.
